velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -478,8 +478,12 @@ void WindowFuzzer::go() {
           generateSortingKeysAndOrders("s", argNames, argTypes, false, numKeys);
     }
 
+    std::vector<std::string> sortingKeys;
```

Comment:
Do you really need this to be a vector and then change it to a set in the function ? We can build an unordered_set here itself.